lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -143,37 +144,45 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
 
     private async getDestinationFolderNodeId(): Promise<string> {
         let rootNodeId: string;
-        let destinationFolderPath = <DestinationFolderPathModel> { alias: AttachFileCloudWidgetComponent.ALIAS_USER_FOLDER, path: '' };
-        if (this.isAlfrescoAndLocal() && this.hasDestinationFolder()) {
-            if (this.isPathVariableType(DestinationFolderPathType.STRING_TYPE) || this.isPathStaticType()) {
-                destinationFolderPath = this.getAliasAndRelativePathFromDestinationFolderPath(this.field.params.fileSource.destinationFolderPath.value);
-                destinationFolderPath.path = this.replaceAppNameAliasWithValue(destinationFolderPath.path);
-            }
-
-            if (this.isPathVariableType(DestinationFolderPathType.FOLDER_TYPE)) {
-                rootNodeId = this.field.params.fileSource.destinationFolderPath.value;
-            }
-        }
 
-        if (!rootNodeId) {
-            try {
-                const nodeId = await this.getNodeIdBasedOnPath(destinationFolderPath);
-                rootNodeId = nodeId ? nodeId : destinationFolderPath.alias;
-            } catch (error) {
-                this.logService.error(error);
-            }
+        switch (this.field?.params?.fileSource?.destinationFolderPath?.type) {
+            case DestinationFolderPathType.STATIC_TYPE:
```

Comment:
Since STRING_TYPE and STATIC_TYPE have the same behaviour you can do an OR
```
 case DestinationFolderPathType.STATIC_TYPE || DestinationFolderPathType.STRING_TYPE:
```

Reply:
I believe. OR operand would not work as expected. Using it with a switch case is not a good practice

switch a {
  case 1,2:  // Noncompliant; only 2 is ever handled by this case
    doTheThing(a);
  case 3 || 4: // Noncompliant; only '3' is handled
    doThatThing(a);
  case 5:
    doTheOtherThing(a);
  default:
    console.log("Neener, neener!");  // this happens when a==1 or a == 4
}

Reply:
ok no problem, thnx for checking